migaloo/assetlist.json

```diff
@@ -572,6 +572,31 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/FROG.png"
         }
       ]
+    },
+    {
+      "description": "GRAC",
```

Comment:
The title of the PR has a better description than the actual description property??

Reply:
You are definitely right, we'll improve that